### PR TITLE
Comb Sort in Kotlin

### LINKED ISSUE #2135: Comb Sort
- [x] C
- [x] C++
- [ ] C# - @sakshijainn
- [ ] CoffeeScript
- [ ] Dart
- [x] Go
- [x] Java
- [ ] JS - @atrip10
- [x] Kotlin - @sandhyabhan
- [x] Php
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by sandhyabhan**
Fixes #2135 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Add Comb Sort In Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Comb_Sort.kt

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Comb_Sort/Comb_Sort.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        }
+        int n = arrSize;
+        obj.combSort(arr, n);
+
+        println("Array after sorting:");
+        for (i in 0 until arrSize) 
+	{
+	    println(arr[i] + " ");
+	}
+
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:59]: Remove line 59 to 61

File: Comb_Sort/Comb_Sort.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        for(i in 0 until arrSize)
+   	{
+       	    arr[i] = read.nextLine().toInt()
+        }
+        int n = arrSize;
+        obj.combSort(arr, n);
+
+        println("Array after sorting:");
+        for (i in 0 until arrSize) 
+	{
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:56]: Inconsistency in indentation

File: Comb_Sort/Comb_Sort.kt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    fun main()
+    { 
+        CombSort obj = new CombSort(); 
+	var read = Scanner(System.`in`)
+    	println("Enter the size of Array:")
+    	val arrSize = read.nextLine().toInt()
+    	var arr = IntArray(arrSize)
+        println("Enter elements")
+        for(i in 0 until arrSize)
+   	{
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:48]: Inconsistency in indentation

File: Comb_Sort/Comb_Sort.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    flag = 1; 
+                } 
+            } 
+        }
+    } 
+
+    // Main function
+    fun main()
+    { 
+        CombSort obj = new CombSort();
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:41]: Inconsistency in indentation

File: Comb_Sort/Comb_Sort.kt [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    // Function to sort arr[] using Comb Sort 
+    fun combSort(int:arr[], int:n):void
+    { 
+        // initialize gap value to array length
+        int gap = n;
+        int flag = 1;
+        while (gap > 1 || flag == 1 ) 
+        { 
+            // Update gap value by using shrink factor 1.3
+            gap = (gap*10) / 13;
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:23]: Add space around *

File: Comb_Sort/Comb_Sort.kt [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+    // Main function
+    fun main()
+    { 
+        CombSort obj = new CombSort(); 
+	var read = Scanner(System.`in`)
+    	println("Enter the size of Array:")
+    	val arrSize = read.nextLine().toInt()
+    	var arr = IntArray(arrSize)
+        println("Enter elements")
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:46]: Inconsistency in indentation

File: Comb_Sort/Comb_Sort.kt (hunk 2 above)
Comment (somya-kapoor) [reply Comb_Sort/Comb_Sort.kt:56]: Inconsistency in indentation

File: Comb_Sort/Comb_Sort.kt (hunk 6 above)
Comment (somya-kapoor) [reply Comb_Sort/Comb_Sort.kt:46]: Inconsistency in indentation

File: Comb_Sort/Comb_Sort.kt [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+       	    arr[i] = read.nextLine().toInt()
+        }
+        int n = arrSize;
+        obj.combSort(arr, n);
+
+        println("Array after sorting:");
+        for (i in 0 until arrSize) 
+	{
+	    println(arr[i] + " ");
+	}
```
Comment (somya-kapoor) [inline Comb_Sort/Comb_Sort.kt:58]: Inconsistency in indentation

**Comment by sandhyabhan** [comment]
I cannot see any of the above changes.
I don't know why.
The code looks good to me.

**Comment by singh-shreya6** [comment]
All of your files have such inconsistencies in space. Use some other text editor.

**Comment by sandhyabhan** [comment]
I tried other code editor and made some changes.

**Pull request merged**